pyia/data.py

```diff
@@ -223,12 +226,36 @@ def pm(self):
         """2D proper motion. Has shape `(nrows, 2)`"""
         _u = self.pmra.unit
         return np.vstack((self.pmra.value, self.pmdec.to(_u).value)).T * _u
+    
+    @u.quantity_input(min_parallax=u.mas, equivalencies=u.parallax())
+    def get_distance(self, min_parallax=None, allow_negative=True):
```

Comment:
I would prefer to keep the default as False, but I do like the idea of adding this argument!
```suggestion
    def get_distance(self, min_parallax=None, allow_negative=False):
```